app/views/taxa/taxonomy_details.html.haml

```diff
@@ -0,0 +1,65 @@
+- content_for :extrajs do
+  :javascript
+    $( document ).ready( function() {
+      var taxon_framework_table = $( '.taxon_framework_table' );
+      taxon_framework_table.hide();
+      $( ".taxon_framework_count" ).find( 'a[href="#"]' ).click( function ( e ) {
+        e.preventDefault();
+        taxon_framework_table.slideToggle( 500 );
+      } );      
+      var flagged_taxa_table = $( '.flagged_taxa_table' );
+      flagged_taxa_table.hide();
+      $( ".flagged_taxa_count" ).find( 'a[href="#"]' ).click( function ( e ) {
+        e.preventDefault();
+        flagged_taxa_table.slideToggle( 500 );
+      } );      
+      $( '[data-toggle="popover"]' ).popover({html:true});
+    } );
+- content_for( :extracss ) do
+  :css
+    .table.table-bordered{
+      border:0px;
+    }
+    .table.table-bordered > thead > tr > th{
+      border:0px;
+      border-bottom:1px solid gray;
+    }
+    .table.table-bordered > tbody > tr > td{
+      border:0px;
+    }
+    .info-link{
+      color: "green";
+    }
+    .media{
+      margin-bottom:5px;
+    }
+    .admin-box {
+        border-width: 2px;
+        border-style: dotted;
+        position: relative;
+        padding: 5px;
+    }
+.container
+  .col-md-12
+    %ul.list-unstyled
+      %li
+        = link_to t(:back_to_x, :noun => @taxon.name), @taxon, :class => "back crumb"
+    %h3
+      = t(:taxonomy_details_for)
```

Comment:
Avoid translation keys that assume word order like this. A better approach is something like `t(:taxonomy_details_for_taxon_html, taxon: render( partial: "shared/taxon", object: @taxon, locals: { link_url: url_for( @taxon ) } ) )` and the translation in English would be `Taxonomy Details for %{taxon}`. That way translators can specify where the the taxon should go in their language, especially important for right-to-left languages, but potentially important for all languages.